iam-drift-detection/detect_drift.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
```

Comment:
nit: prefer using a pure posix shell to be more portable. I don't see anything bash specific in this file.

```suggestion
#!/bin/sh
```

Reply:
Done.